hummingbot/logger/report_aggregator.py

```diff
@@ -0,0 +1,186 @@
+import asyncio
+import copy
+import logging
+
+from hummingbot.logger import REPORT_EVENT_QUEUE
+from hummingbot.logger.struct_logger import StructLogger
+from collections import defaultdict
+from typing import Optional
+from hummingbot.core.utils.async_utils import safe_ensure_future
+from hummingbot.core.utils.exchange_rate_conversion import ExchangeRateConversion
+from hummingbot.market.bamboo_relay.bamboo_relay_market import BambooRelayMarket
+from hummingbot.market.binance.binance_market import BinanceMarket
+from hummingbot.market.bittrex.bittrex_market import BittrexMarket
+from hummingbot.market.coinbase_pro.coinbase_pro_market import CoinbaseProMarket
+from hummingbot.market.ddex.ddex_market import DDEXMarket
+from hummingbot.market.huobi.huobi_market import HuobiMarket
+from hummingbot.market.liquid.liquid_market import LiquidMarket
+from hummingbot.market.idex.idex_market import IDEXMarket
+from hummingbot.market.radar_relay.radar_relay_market import RadarRelayMarket
+from hummingbot.market.dolomite.dolomite_market import DolomiteMarket
+from hummingbot.market.bitcoin_com.bitcoin_com_market import BitcoinComMarket
+from hummingbot.market.hitbtc.hitbtc_market import HitBTCMarket
+
+MARKETS = {
+    "ddex": DDEXMarket,
+    "coinbase_pro": CoinbaseProMarket,
+    "binance": BinanceMarket,
+    "bamboo_relay": BambooRelayMarket,
+    "radar_relay": RadarRelayMarket,
+    "idex": IDEXMarket,
+    "dolomite": DolomiteMarket,
+    "huobi": HuobiMarket,
+    "liquid": LiquidMarket,
+    "bittrex": BittrexMarket,
+    "bitcoin_com": BitcoinComMarket,
+    "hitbtc": HitBTCMarket
+}
+
+
+class ReportAggregator:
```

Comment:
We have chose to remove this class, since we no longer see a need for it. Remove it from this PR.